index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p class="nav-logo">john doe</p>
+        <button class="nav-btn">
+          <i class="fa fa-bars" aria-hidden="true"></i>
+        </button>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="home-section">
+        <h1 class="section-title">Hey There. i'm Alex</h1>
+        <article class="info-container">
+          <h2 class="info-title">I'am a Software Developer</h2>
+          <p class="info">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hestiate to contact me.
+          </p>
+        </article>
+        <div class="links-container">
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/git.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/linkedin.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/angelist.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/twitter.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/medium.svg"
+          /></a>
+        </div>
```

Comment:
- [ ] Since the contents of this element are a list of links, it will be semantically more convenient to structure them using `<ul>` and `<li>` elements. You can replace the `<div>` with a `<ul>` tag. 😎️
The final structure could look like this
```
<ul class="links-container">
  <li>
    <a><img/></a>
  </li>
  <li>
    <a><img/></a>
  </li>
  ...
</ul>
```